Client/Frontend/Widgets/Themes/Theme.swift

```diff
@@ -6,312 +6,35 @@ import Foundation
-            let accentStr = try container.decode(String.self, forKey: .accent)
-
-            header = UIColor(colorString: headerStr)
-            footer = UIColor(colorString: footerStr)
-            home = UIColor(colorString: homeStr)
-            addressBar = UIColor(colorString: addressBarStr)
-            border = UIColor(colorString: borderStr)
-            accent = UIColor(colorString: accentStr)
-            
-            stats = try container.decode(Stat.self, forKey: .stats)
-            tints = try container.decode(Tint.self, forKey: .tints)
-            transparencies = try container.decode(Transparency.self, forKey: .transparencies)
-        }
-        
-        let header: UIColor
-        let footer: UIColor
-        let home: UIColor
-        let addressBar: UIColor
-        let border: UIColor
-        let accent: UIColor
-        
-        let stats: Stat
-        struct Stat: Decodable {
-            init(from decoder: Decoder) throws {
-
-                let container = try decoder.container(keyedBy: ThemeCodingKeys.ColorCodingKeys.StatCodingKeys.self)
-                let adsStr = try container.decode(String.self, forKey: .ads)
-                let trackersStr = try container.decode(String.self, forKey: .trackers)
-                let dataSavedStr = try container.decode(String.self, forKey: .dataSaved)
-                let timeSavedStr = try container.decode(String.self, forKey: .timeSaved)
-
-                ads = UIColor(colorString: adsStr)
-                trackers = UIColor(colorString: trackersStr)
-                dataSaved = UIColor(colorString: dataSavedStr)
-                timeSaved = UIColor(colorString: timeSavedStr)
-            }
-
-            let ads: UIColor
-            let trackers: UIColor
-            let dataSaved: UIColor
-            let timeSaved: UIColor
-        }
-        
-        let tints: Tint
-        struct Tint: Decodable {
-            init(from decoder: Decoder) throws {
-
-                let container = try decoder.container(keyedBy: ThemeCodingKeys.ColorCodingKeys.TintCodingKeys.self)
-                let homeStr = try container.decode(String.self, forKey: .home)
-                let headerStr = try container.decode(String.self, forKey: .header)
-                let footerStr = try container.decode(String.self, forKey: .footer)
-                let addressBarStr = try container.decode(String.self, forKey: .addressBar)
-
-                home = UIColor(colorString: homeStr)
-                header = UIColor(colorString: headerStr)
-                footer = UIColor(colorString: footerStr)
-                addressBar = UIColor(colorString: addressBarStr)
-            }
-
-            let home: UIColor
-            let header: UIColor
-            let footer: UIColor
-            let addressBar: UIColor
+enum DefaultTheme: String, RepresentableOptionType {
+    case system = "Z71ED37E-EC3E-436E-AD5F-B22748306A6B"
```

Comment:
We leave these json files and Theme class?

Reply:
The JSON files were removed, but unfortunately I need to keep these UUID's for system/light/dark as they are the values that are saved in NSUserDefaults